src/Maestro/DependencyUpdateErrorProcessor/DependencyUpdateErrorProcessor.cs

```diff
@@ -242,10 +255,10 @@ public async Task ProcessDependencyUpdateErrorsAsync()
                                 break;
                             }
 
-                            _logger.LogInformation($"Matching issue {issueNumber} for subscription {subscriptionUpdateHistoryError.SubscriptionId} was closed. Creating a new issue.");
+                            _logger.LogInformation("Matching issue {IssueNumber} for subscription {SubscriptionId} was closed. Creating a new issue.", issueNumber.Value, subscriptionUpdateHistoryError.SubscriptionId);
```

Comment:
does String() need to be called here? SubscriptionId is a Guid.

Reply:
Much like the interpolated thing, it's going to ToString, everything. Though GUID's have lots of formats, so it probably makes sense to call ToString yourself to get the format you want, and not let it just be whatever C# feels like.

Reply:
Fine by me then. Just making sure we didn't get `System.Guid`

Reply:
Its default ToString uses the "D" specifier, which gives a pleasant form like `382c74c3-721d-4f34-80e5-57657b6cbc27`